2_observations/src/data_munge_functions.R

```diff
@@ -49,11 +49,6 @@ filter_temp_data_to_sites <- function(sites_ind, dat_ind, out_ind){
 
   drb_dat <- filter(dat, site_id %in% unique(sites$site_id)) %>%
```

Comment:
The output from `filter_temp_data_to_sites()` was meant to represent the raw temperature data matched to the NHM segments that we could archive or send to someone that was interested in the unaggregated data. 

These changes look great, but I also want to make sure the target that gets built using this function (`2_observations/out/obs_temp_drb_raw.rds.ind`) still serves us in this way. For example, it seems like we could move the code to reconcile the sub-locations (lines 83-109 below) from `munge_split_temp_dat()` to `filter_temp_data_to_sites()` if it would make more sense to archive the unaggregated data that has been filtered for multiple sub-locations/site-dates. I'm not advocating for this change, just trying to think through how these targets will be used. What do you think?

Reply:
I think it makes sense to leave the raw data as raw as we can while still making it useful. I would advocate for leaving it as-is for the following reasons:

- we're aggregating at multiple levels; both the site level and the reach level, and so the raw data represents the unaggregated version of both of those levels.
- I can imagine a scenario where the unaggregated `sub_location`s are useful -- e.g., if you did want to look at the piezometer data or if there were multiple depths and you were interested in stratification. Leaving it raw gives the user more flexibility and the opportunity to dig into those sub-locations do decide which is best for their use case.